src/components/ListV2/ListLoadingPlaceholder.tsx

```diff
@@ -0,0 +1,39 @@
+import styled from '@emotion/styled'
+import Line from '../Placeholder/Line'
+import { ListCell } from './ListCell'
+import { ListRow } from './ListRow'
+
+const StyledPlaceholder = styled(Line)`
+  width: 80%;
+`
+
+type ListLoadingPlaceholderProps = {
+  rows?: number
+  cols?: number
+  className?: string
+}
+
+export const ListLoadingPlaceholder = ({
+  cols = 1,
+  rows = 5,
+  className,
+}: ListLoadingPlaceholderProps) => (
+  <>
```

Comment:
useless no ?

Reply:
No because it need to be typed as JSX.Element and not JSX.Element[] so I did this to avoid as JSX.Element